### ISSUE TITLE
_header.scss missing color variable

### ISSUE THREAD

**Issue by JoomFX** [gantry/gantry5#48]
If you check the _header.scss file (g5_hydrogen\scss\hydrogen) you will see the following:

```
#g-header {
	background: $header-background;
}
```

Isn't it supposed to be:

```
#g-header {
	background: $header-background;
	color: $header-text-color;
}
```

Or you've taken out the "color" on purpose?

**Comment by kevlardev** [comment]
Yes, this is actually already fixed with 8da16514b1998962b2420592c6340b14165fc038

**Comment by JoomFX** [comment]
Thanks :)
